src/app/component/eco-news/create-news/create-news.component.css

```diff
@@ -205,6 +209,10 @@ button {
     border: none;
 }
 
+.red {
```

Comment:
please change the name of class to more understandable

Reply:
fixed